### PR TITLE
Feature/automated testing fix standalone testing

### PR DESCRIPTION
**Pull request by markusstuhler**

### CODE REVIEW

**Review by dacolgit — APPROVED** [review]

File: e2e-tests/IIoTPlatform-E2E-Tests/TestExtensions/IIoTPlatformTestContext.cs [hunk 1]
```diff
@@ -191,6 +191,6 @@ public void Dispose()
             () => string.Empty);
 
         string IContainerRegistryConfig.ImagesTag => GetStringOrDefault(TestConstants.EnvironmentVariablesNames.PCS_IMAGES_TAG,
-            () => throw new Exception("Images tag not provided"));
+            () => "mcr.microsoft.com/iotedge/opc-publisher:latest" );
```
Comment (karok2m) [inline e2e-tests/IIoTPlatform-E2E-Tests/TestExtensions/IIoTPlatformTestContext.cs:194]: Should be just `latest`.

Comment (markusstuhler) [reply e2e-tests/IIoTPlatform-E2E-Tests/TestExtensions/IIoTPlatformTestContext.cs:194]: It is the whole image when the ContainerServer is not set...

File: e2e-tests/IIoTPlatform-E2E-Tests/Orchestrated/PublishSingleNodesTestTheory.cs [hunk 2]
```diff
@@ -289,5 +289,16 @@ public class PublishSingleNodesTestTheory
                 _output.WriteLine($"ErrorMessage: {response.ErrorMessage}");
             }
         }
+
+        [Fact, PriorityOrder(12)]
+        public async void Test_VerfiyNoDataIncomingAtIoTHub() {
+            await Task.Delay(90 * 1000); //wait till the publishing has stopped
+            //use test event processor to verify data send to IoT Hub
+            await TestHelper.StartMonitoringIncomingMessages(_context, 0, 0, 0);
```
Comment (koepalex) [inline e2e-tests/IIoTPlatform-E2E-Tests/Orchestrated/PublishSingleNodesTestTheory.cs:297]: Why not use the new overload, were only pass test context?

Comment (markusstuhler) [reply e2e-tests/IIoTPlatform-E2E-Tests/Orchestrated/PublishSingleNodesTestTheory.cs:297]: I didn't know of it. ;-)

**Review by koepalex — APPROVED** [review]
:shipit:

File: e2e-tests/IIoTPlatform-E2E-Tests/TestExtensions/IIoTPlatformTestContext.cs (hunk 1 above)
Comment (karok2m) [reply e2e-tests/IIoTPlatform-E2E-Tests/TestExtensions/IIoTPlatformTestContext.cs:194]: No, I it should be only the tag. Here is the image URI generation in `IoTHubPublisherDeployment.cs` where we use the tag:

```csharp
            var version = _context.ContainerRegistryConfig.ImagesTag ?? "latest";
            var image = $"{server}/{ns}iotedge/opc-publisher:{version}";
```

**Pull request merged**